modules/tensor_mechanics/doc/content/modules/tensor_mechanics/ShellElements.md

```diff
@@ -0,0 +1,78 @@
+# Shell elements
+
+A shell element is used to model the response of structural elements that are much thinner along one direction (out-of-plane direction) compared to the two in-plane directions. A 4-node shell element is implemented in Moose based on [!cite](dvorkin1984continuum) that can model structural response of both thin and thick plates.
+
+!media media/tensor_mechanics/shell_representation.png
+      style=width:50%;margin-left:2%;float:right
+      id=fig_shell
+      caption=Shell element with 4 nodes and 3 translational and 2 rotational degrees of freedom at each node.
+
+Each of the four nodes have 5 degrees of freedom - 3 displacements ($u_1^k$,$u_2^k$, $u_3^k$) and 2 rotations ($\alpha_k$ and $\beta_k$). The thickness ($a_k$) is assumed to be constant across the shell element and the initial normal ($^oV_n^k) at all nodes are same as the normal to the shell element, which is automatically computed using the coordinates of the nodes.
+
+The basic equation of motion for a quasi-static shell is the same as that of a continuum brick element:
+
+\begin{equation}
+\nabla \cdot \sigma = F_{ext}
+\end{equation}
+
+The quasi-static shell element implementation is Moose is spread across four different objects: strain computation ([ADComputeIncrementalShellStrain](/ADComputeIncrementalShellStrain.md) & [ADComputeFiniteShellStrain](/ADComputeFiniteShellStrain.md)), elasticity tensor computation ([ADComputeIsotropicElasticityTensorShell](/ADComputeIsotropicElasticityTensorShell.md)), stress computation ([ADComputeShellStress](/ADComputeShellStress.md)) and stress divergence kernel ([ADStressDivergenceShell](/ADStressDivergenceShell.md)).
+
+
+## Strain Calculation
+
+The first step to calculating stresses is to calculate the strains in the shell element. All the computations are performed in the natural coordinate system of the shell ($r_1$, $r_2$ and $r_3$). The geometry of the element at any time t is defined as:
+
+\begin{equation}
+^t x_i =  \sum_{k=1}^4 h_k ^t {x_i}^k + \frac{r_3}{2} \sum_{k=1}^4 a_k h_k ^tV_{ni}^k
+\end{equation}
+
+Therefore, the incremental displacements of any point with natural coordinate $r_i$ of the shell element at time $t$ in the global cartesian coordinate system ($^ox_1$, $^ox_2$ and $^ox_3$) are:
```

Comment:
'any point with natural coordinate $r_i$ of the shell element' -> 'any point within the shell element with natural coordinate $r_i$'